### ISSUE TITLE
How do I use multiple connectors for create_engine ? 

### ISSUE THREAD

**Issue by Gowtham95india** [dropbox/PyHive#168]
I have defined tables like the general way in sqlalchemy. 

Problem is I have few tables in one connector and other tables in another connectors. 

Is it possible to use multiple connectors in single instance of create engine? 

class VMarg(object): __table_args__ ={'schema': 'backend'}

class Users(VMarg, Base): __tablename__ = 'users'

Schema/Database backend is in one MySQL instance which defined in connector vmarg.
The same way orders is defined. And Orders is available under orders connector. 

If I don't defined connector in connection string, by default it is taking hive and is continuing to operate under hive. 

How can we define different connectors in single create engine ?

**Comment by jingw** [comment]
I don't know, sorry. I think you'll have better luck looking around the SQLAlchemy documentation.

I'm not familiar with SQLAlchemy's ORM stuff, but SQLAlchemy's `Table` constructor takes a `schema` argument, e.g. https://github.com/dropbox/PyHive/blob/master/pyhive/tests/test_sqlalchemy_hive.py#L175

**Comment by Gowtham95india** [comment]
@jingw, you are correct. If you look the post, I already used schema in table construction. But somehow we need to tell that, you have use this catalog for this table to presto from sqlalchemy. 

So that it uses that particular catalog.

**Comment by AlexandreOuellet** [comment]
Created pull request in #246.  This is enough for my use case (simply being able to query), and hopefully will be useful to you as well.  Usage is to put presto_catalog='my_catalog' within your models' definition
